msal4j-sdk/src/integrationtest/java/com.microsoft.aad.msal4j/ClientCredentialsIT.java

```diff
@@ -160,7 +160,7 @@ private ClientAssertion getClientAssertion(String clientId) {
                 clientId,
                 (ClientCertificate) certificate,
                 "https://login.microsoftonline.com/common/oauth2/v2.0/token",
-                true);
+                true, false);
```

Comment:
Am I correct in understanding that the existing tests cover the following?

- AAD + sha2
- ADFS + sha1